packages/account-abstraction/package.json

```diff
@@ -39,7 +39,9 @@
   },
   "dependencies": {
     "@gnosis.pm/safe-contracts": "^1.3.0",
+    "@gnosis.pm/safe-deployments": "^1.19.0",
```

Comment:
Remember to update to latest version

Reply:
This package is also duplicated in `devDependencies`

`prettier` and `typescript` are already available at the root `package.json` so they should not be needed here

Reply:
Thanks @dasanra. Changes were applied